### PR TITLE
Fix prerequisites

### PR DESCRIPTION
**Pull request by gdelafond**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ Compilation
 
 Cloud Gateway's build is based on `cmake`.
 
+
+Install prerequisites:
+
+```
+sudo apt-get install libz-dev libssl-dev
```
Comment (rgacogne) [inline README.md]: so, why are those not included in the prerequisites above?

Comment (gdelafond) [reply README.md]: They are not needed to build libevaio. But we can put them with the previous one.

Comment (rgacogne) [reply README.md]: Ah, it was not clear to me that the previous ones were only related to `libevaio`. I think it would be easier to group them, yes.

Comment (gdelafond) [reply README.md]: done

**Review by rgacogne — APPROVED** [review]

**Pull request merged**